### DISCUSSION TITLE
新手友好的翻译过的学习笔记

### DISCUSSION THREAD

**Discussion by zedz** [IntensiveCoLearning/Web3-URL#123] [Show and tell]
我的时间可能多一点，基础弱一点，花了些心思做了这个笔记本。
希望对有同样需求的同学有帮助。

里面有通过chatGPT学习的记录
有对[https://docs.web3url.io/](https://docs.web3url.io/)的中文翻译
以及新手可能不了解的名词的chatGPT给出的通俗解释

目前更新到7.20，感觉有点开始变难，发出来希望可以倒逼学习。

[新手友好 - WEB3URL学习笔记](https://gigantic-bandicoot-5c1.notion.site/Web3URL-01270b246e7f4309afd927b3c9764ec1)

**Comment by luffythink** [comment]
很棒，好像 notion 链接有权限打不开。

**Reply by zedz** [reply]
谢谢提醒！好像现在可以了，好像需要publish才能不需要权限，调整了一下。
